doc/src/modules/polys/basics.rst

```diff
@@ -14,6 +14,319 @@ polynomials within SymPy. All code examples assume::
     >>> x, y, z = symbols('x,y,z')
     >>> init_printing(use_unicode=False, wrap_line=False, no_global=True)
 
+Basic concepts
+==============
+
+Polynomials
+-----------
+
+Given a family `(x_i)` of symbols, or other suitable objects,
+expressions derived from them by repeated addition, subtraction
+and multiplication are called *polynomial expressions in the
+generators* `x_i`.
+
+By the distributive law it is possible to perform
+multiplications before additions and subtractions.
+The products of generators thus obtained are called
+*monomials*. They are usually written in the form
+`x_1^{\nu_1}x_2^{\nu_2}\cdots x_n^{\nu_n}` where the exponents `\nu_i`
+are nonnegative integers. It is often convenient to write this briefly
+as `x^\nu` where `x = (x_1, x_2, \ldots, x_n)` denotes the family of
+generators and `\nu = (\nu_1, \nu_2, \ldots, \nu_n)` is the
+family of exponents.
+
+When all monomials having the same exponents are combined, the polynomial
+becomes a sum of products `c_\nu x^\nu`, called the *terms* of the polynomial,
+where the *coefficients* `c_\nu` are integers.
```

Comment:
This is about 'polynomial expressions'. The concepts of 'coefficient' and 'generator' can be exactly defined only for actual 'polynomials' Then coefficients can be elements of any commutative ring.

 For example, the polynomial expression `x*y - 2` can be transformed into a polynomial in several ways. 2 will always be a coefficient, but `y` can also be a coefficient in the ring `ZZ[y][z]` of polynomials in `x` with coefficients in `ZZ[y]`.